custom_components/test/fuzzing/grammar/test_gram_on_xxx.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+from custom_components.test.fuzzing.fuzzer_utils.GrammarFuzzer import GrammarFuzzer
+from custom_components.test.fuzzing.fuzzer_utils.grammars.grammar_ipv4 import grammar_ipv4
+
+logger = logging.getLogger(__name__)
+test_fuzzer = GrammarFuzzer()
+
+def test_dummy() -> None:
```

Comment:
What does a test case and/or a fuzzer look like now? How do we deal with this if the function under test requires an `int` (e.g. from the `ValuePoolFuzzer`) and a `str` from the `GrammarFuzzer`?